### PR TITLE
add esdl messages

### PR DESCRIPTION
**Pull request by MarkTNO**

### CODE REVIEW

File: src/omotes_orchestrator/main.py [hunk 1]
```diff
@@ -13,13 +13,15 @@
 from omotes_sdk.internal.orchestrator_worker_events.messages.task_pb2 import (
     TaskResult,
     TaskProgressUpdate,
+    TaskEsdlMessage,
```
Comment (cwang39403) [inline src/omotes_orchestrator/main.py:16]: I wondered if it is possible and reasonable to define one common EsdlMessage protocol in SDK and reuse it in different places. As TaskEsdlMessage and EsdlMessage have the same definition. Maybe with import? https://protobuf.dev/programming-guides/proto3/#importing

File: src/omotes_orchestrator/main.py [hunk 2]
```diff
@@ -453,6 +455,16 @@ def task_result_received(self, task_result: TaskResult) -> None:
                 self._init_barriers.wait_for_barrier(job.id)
                 job_db = self.postgresql_if.get_job(job.id)
 
+            esdl_messages_job = [
```
Comment (cwang39403) [inline src/omotes_orchestrator/main.py:458]: nitpicking, `job_esdl_messages`?

Comment (MarkTNO) [reply src/omotes_orchestrator/main.py]: not used anymore

**Review by lfse-slafleur — APPROVED** [review]

**Comment by lfse-slafleur** [comment]
Approved assuming tests, typecheck & integration test are still fixed after updating SDK.

**Review by cwang39403 — COMMENTED** [review]
While these connection errors from integration tests are also shown in the earlier successful pull requests, I am unsure if this is the expected behavior. @lfse-slafleur perhaps you have some clues?

image

**Comment by MarkTNO** [comment]
> While these connection errors from integration tests are also shown in the earlier successful pull requests, I am unsure if this is the expected behavior. @lfse-slafleur perhaps you have some clues?
> 
> image

As we discussed yesterday Cheng-Kai, I also see these errors when I run locally.

**Comment by lfse-slafleur** [comment]
> While these connection errors from integration tests are also shown in the earlier successful pull requests, I am unsure if this is the expected behavior. @lfse-slafleur perhaps you have some clues?
> 
@cwang39403 The workers try to connect to RabbitMQ but RabbitMQ isn't ready yet so it retries to connect until it succeeds. The errors aren't really fatal errors in that regard as they are expected. 

I am unsure if it is possible, but if you add the 'healthy' condition check as the other workers do (https://github.com/Project-OMOTES/omotes-system/blob/2eb6c9e39903d2337fca22dac87c67951bdaf4b8/docker-compose.yml#L154) then it should be remove the errors for retrying to connect.

**Comment by cwang39403** [comment]
> > While these connection errors from integration tests are also shown in the earlier successful pull requests, I am unsure if this is the expected behavior. @lfse-slafleur perhaps you have some clues?
> 
> @cwang39403 The workers try to connect to RabbitMQ but RabbitMQ isn't ready yet so it retries to connect until it succeeds. The errors aren't really fatal errors in that regard as they are expected.
> 
> I am unsure if it is possible, but if you add the 'healthy' condition check as the other workers do (https://github.com/Project-OMOTES/omotes-system/blob/2eb6c9e39903d2337fca22dac87c67951bdaf4b8/docker-compose.yml#L154) then it should be remove the errors for retrying to connect.

Thanks for the clarification! @lfse-slafleur @MarkTNO

**Pull request merged**